### ISSUE TITLE
Building and Packaging?

### ISSUE THREAD

**Issue by ediathome** [pioz/neutron-ruby-electron#1]
Hi pioz!
This project looks really interesting. Is there a way to build and package the applications created with Neutron?
Cheers
ediathome

**Comment by pioz** [comment]
No, at this time there aren't help tools to build and create a package of your app. 😔
Any help on this is really appreciated btw... 😅

**Comment by ediathome** [comment]
Well two options come to my mind for building:
1. Compiling Ruby to JS with http://opalrb.com
2. Compiling Ruby to executable with something like https://github.com/larsch/ocra/
What's your opinion on these?
